src/sentry/apidocs/hooks.py

```diff
@@ -197,18 +198,44 @@ def custom_postprocessing_hook(result: Any, generator: Any, **kwargs: Any) -> An
     for path, endpoints in result["paths"].items():
         for method_info in endpoints.values():
             _check_tag(path, method_info)
+            endpoint_name = f"'{method_info['operationId']}'"
 
             if method_info.get("description") is None:
                 raise SentryApiBuildError(
-                    "Please add a description to your endpoint method via a docstring"
+                    f"Please add a description via docstring to your endpoint {endpoint_name}"
                 )
-            # ensure path parameters have a description
+
             for param in method_info.get("parameters", []):
+                # Ensure path parameters have a description
                 if param["in"] == "path" and param.get("description") is None:
                     raise SentryApiBuildError(
-                        f"Please add a description to your path parameter '{param['name']}'"
+                        f"Please add a description to your path parameter '{param['name']}' for endpoint {endpoint_name}"
                     )
 
+            # Ensure body parameters are sorted by placing required parameters first
+            if "requestBody" in method_info:
+                try:
+                    content = method_info["requestBody"]["content"]
+                    # media type can either "multipart/form-data" or "application/json"
+                    if "multipart/form-data" in content:
+                        schema = content["multipart/form-data"]["schema"]
+                    else:
+                        schema = content["application/json"]["schema"]
+
+                    # Required params are stored in a list and not in the param itself
+                    required = set(schema.get("required", []))
+                    if required:
+                        # Explicitly sort body params by converting the dict to an ordered dict
+                        schema["properties"] = OrderedDict(
```

Comment:
I think explicitly using an `OrderedDict` is better than sorting the original dict and relying on our python version having implicit ordering